back/mail-processing/request_daily_idea.py

```diff
@@ -4,12 +4,14 @@
 from utils.models import UserModel
 from mail_templates.request_daily_idea.send_request_daily import send_daily_bulk
 import datetime
+import sentry_sdk
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
 
+sentry_sdk.init(dsn=os.environ.get('SENTRY_DSN'), integrations=[AwsLambdaIntegration()])
```

Comment:
there no import of os - and so hence it will fail at initialize please check all files for correct imports

Reply:
Yeah, missed it, my IDE doesn't read Python, probably need to install Pycharm